scheduler/ecs/ecs.go

```diff
@@ -424,7 +397,83 @@ func (m *ecsProcessManager) updateCreateService(ctx context.Context, app *schedu
 	return m.createService(ctx, app, p)
 }
 
-func (m *ecsProcessManager) Processes(ctx context.Context, appID string) ([]*scheduler.Process, error) {
+func (m *Scheduler) loadBalancer(ctx context.Context, app *scheduler.App, p *scheduler.Process) (string, error) {
```

Comment:
Nothing has actually changed within this method, it's identical to the old `LBProcessManager.CreateProcess` method.